hazelcast/src/main/java/com/hazelcast/security/IPermissionPolicy.java

```diff
@@ -46,6 +48,8 @@
      */
     PermissionCollection getPermissions(Subject subject, Class<? extends Permission> type);
 
+    boolean refreshClientPermissions(Set<PermissionConfig> permissionConfigs);
```

Comment:
Why not `refreshPermissions()`? `getPermissions()` method doesn't contain `client`.